CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4 (07 October 2016)
+
+* Add ability to remove repository location (#86)
+* Fixed bug in connection parsing when federated connections are present (#87)
```

Comment:
We also had a new contributor fix a Unicode bug in python: https://github.com/tableau/document-api-python/pull/80